pkgs/applications/misc/ff2mpv/default.nix

```diff
@@ -21,7 +21,7 @@ stdenv.mkDerivation rec {
 
   installPhase = ''
```

Comment:
Hope you don't mind some extra suggestions, since it's a refactor and all.

```suggestion
  installPhase = ''
    runHook preInstall
```

Reply:
Done.